packages/types/src/starknet/interfaces.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import {BlockFilter} from '@subql/types-core';
+import {TransactionReceipt} from 'starknet';
+import {BLOCK_HASH, BLOCK_NUMBER, FELT, RESOURCE_PRICE, TXN_TYPE, BLOCK_STATUS} from './rpcSpec';
```

Comment:
The rpcSpec can be removed and this used instead

```ts
import {
  BlockHash as BLOCK_HASH,
  Felt as FELT,
  SPEC,
} from 'starknet-types-07';

// The types could be used directly in this file rather than redefining them
type BLOCK_NUMBER = SPEC.BLOCK_NUMBER;
type RESOURCE_PRICE = SPEC.RESOURCE_PRICE;
type TXN_TYPE = SPEC.TXN_TYPE;
type BLOCK_STATUS = SPEC.BLOCK_STATUS;


```

Reply:
Package has been deprecated, I found "@starknet-io/types-js" is the most recent one